pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -323,6 +323,38 @@ func (k *Kubernetes) ConfigServicePorts(name string, service kobject.ServiceConf
 	return servicePorts
 }
 
+// ConfigTmpfs configure the tmpfs.
+func (k *Kubernetes) ConfigTmpfs(name string, service kobject.ServiceConfig) ([]api.VolumeMount, []api.Volume) {
+	volumeMounts := []api.VolumeMount{}
+	volumes := []api.Volume{}
+	var count int
+	var volumeName string
+	for _, volume := range service.TmpFs {
+
+		volumeName = fmt.Sprintf("%s-tmpfs%d", name, count)
+		count++
```

Comment:
no need for count, you can use the range loop for the index value..

`for index, volume := range service.TmpFs {`